visualstimulation/tests/test_analysis.py

```diff
@@ -42,6 +43,24 @@ def test_compute_osi():
     rates = np.array([3, 1, 2, 1, 1, 2, 1, 1])*pq.Hz
     with pytest.warns(UserWarning):
         assert abs(compute_osi(rates, orients) - 1/5) < 1e-12
+    
+    # normalise true
+    orients = np.arange(0, 315, 45)*pq.deg
+    rates = np.array([1, 1, 2, 1, 1, 2, 1, 1])*pq.Hz
+    assert abs(compute_osi(rates, orients, normalise=True) - 1) < 1e-12
```

Comment:
I don't think the tests below are affected by `normalise=True` since `n_rates` is not used